src/reflect/scala/reflect/runtime/JavaUniverseForce.scala

```diff
@@ -245,7 +245,6 @@ trait JavaUniverseForce { self: runtime.JavaUniverse  =>
     definitions.SymbolModule
     definitions.StringAddClass
     definitions.ScalaNumberClass
-    definitions.TraitSetterAnnotationClass
```

Comment:
The compiler won't bootstrap without this file, actually, since it's a required class. I'll deprecate for M5 and we can remove in RC1